internal/access/access_key.go

```diff
@@ -36,15 +36,6 @@ func CreateAccessKey(c *gin.Context, accessKey *models.AccessKey, identityID uid
 		return "", err
 	}
 
-	identity, err := data.GetIdentity(db, data.ByID(identityID))
```

Comment:
The `identityID` parameter to this function is no longer used now that this code is deleted. I think we can remove it.